pmm-app/src/pmm-settings/components/PlatformLogin/SignUp.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { FC } from 'react';
+import { Field, Form, FormRenderProps } from 'react-final-form';
+import { LinkButton, useTheme } from '@grafana/ui';
+import validators from 'shared/components/helpers/validators';
+import { ButtonWithSpinner } from 'shared/components/Form';
+import { showErrorNotification, showSuccessNotification } from 'shared/components/helpers';
+import { Credentials, LoginFormProps } from './types';
+import { Messages } from './PlatformLogin.messages';
+import { CheckboxFieldAdapter, InputFieldAdapter } from './FieldAdapters/FieldAdapters';
+import { getStyles } from './PlatformLogin.styles';
+import { PlatformLoginService } from './PlatformLogin.service';
+import { PRIVACY_POLICY_URL, TERMS_OF_SERVICE_URL } from './PlatformLogin.constants';
+
+export const SignUp = ({ changeMode, getSettings }: LoginFormProps) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  const handleSignUpFormSubmit = async (credentials: Credentials) => {
+    try {
+      await PlatformLoginService.signUp(credentials);
+
+      getSettings();
+      showSuccessNotification({ message: Messages.signUpSucceeded });
+    } catch (e) {
+      console.error(e);
+      showErrorNotification({ message: Messages.errors.signUpFailed });
+    }
+  };
+
+  const CheckboxLabel = () => (
+    <span data-qa="sign-up-agreement-checkbox-label" className={styles.checkboxLabel}>
+      {`${Messages.agreementFirstPart} `}
+      <LinkButton className={styles.link} variant="link" href={TERMS_OF_SERVICE_URL}>
+        {Messages.termsOfService}
+      </LinkButton>
+      {` ${Messages.agreementSecondPart} `}
+      <LinkButton className={styles.link} variant="link" href={PRIVACY_POLICY_URL}>
+        {Messages.privacyPolicy}
+      </LinkButton>
+    </span>
+  );
+
+  const SignUpForm: FC<FormRenderProps<Credentials>> = ({
+    pristine,
+    submitting,
+    valid,
+    handleSubmit,
+  }) => (
+    <form data-qa="sign-up-form" className={styles.form} onSubmit={handleSubmit}>
+      <legend className={styles.legend}>{Messages.perconaPlatform}</legend>
+      <Field
+        data-qa="sign-up-email-input"
+        name="email"
+        label={Messages.emailLabel}
+        component={InputFieldAdapter}
+        validate={validators.compose(
+          validators.required,
+          validators.validateEmail,
+        )}
+      />
+      <Field
+        data-qa="sign-up-password-input"
+        name="password"
+        label={Messages.passwordLabel}
+        type="password"
+        component={InputFieldAdapter}
+        validate={validators.required}
+        autoComplete="on"
+      />
+      <Field
+        data-qa="sign-up-password-input"
+        name="confirm-password"
+        label={Messages.confirmPasswordLabel}
+        type="password"
+        component={InputFieldAdapter}
+        validate={validators.matches('password', Messages.errors.passwordMismatch)}
+        autoComplete="on"
+      />
```

Comment:
this has the same `data-qa` as the previous one. also, was it requested to have a password confirmation here?

Reply:
I'll change data qa, thanks! No, it wasn't, I just added it because I guess that it really makes sense

Reply:
I'd discuss about this also with @atymchuk, since I think the trend now is to have just one password field for sign ups, with no confirmation, but with a "restore password" link.

Reply:
Good point, lets wait for @atymchuk comments, this functionality could be removed few minutes if we need it

Reply:
Yes, one field only for the password.

Reply:
done